packages/transport-http/src/index.js

```diff
@@ -0,0 +1,13 @@
+import { Transport } from '@stellarjs/abstract-transport';
+
+class HttpTransport extends Transport {
+  getHttpMethodAndUrlFromQueueName(queueName) { // eslint-disable-line class-methods-use-this
```

Comment:
I think it can be a utility function that is in use by the two transports..
There is no point in inheriting a static function :)

Reply:
I prefer a static function over a utility, keeps implementation object-oriented.